packages/rocketchat-slashcommands-shrug/shrug.js

```diff
@@ -0,0 +1,20 @@
+/*
+* Shrug is a named function that will replace /shrug commands
+* @param {Object} message - The message object
+*/
+
+
+function Shrug (command, params, item) {
+	if (command === "shrug") {
+		var msg;
+
+		msg = item;
+		msg.msg = params + ' ¯\\_(ツ)_/¯';
+		Meteor.call('sendMessage', msg);
+	}
+}
+
+RocketChat.slashCommands.add('shrug', Shrug, {
+	description: 'Slash_Shrug_Description',
+	params: 'your message (optional)'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2530061390/issues/source?bid=3199634&fileBranchId=3283147#l19)